region/brinstar/pink.json

```diff
@@ -839,19 +852,84 @@
           "from": 1,
           "to": [
             {
-              "id": 3,
+              "id": 4,
               "requires": [
-                "Super",
-                {"cost":[
-                  {"ammo": [
-                    {
-                      "type": "Super",
-                      "count": 1
-                    }
-                  ]}
-                ]}
-               ],
-              "unlock": null
+                {"obstacle":{
+                  "id": "A",
+                  "requires":[
+                    {"or":[
+                      {"cost": [
+                        {"enemyDamage": [{
+                          "enemy": "Sidehopper",
+                          "type": "contact",
+                          "hits": 6
+                        }]}
+                      ]},
+                      {"and":[
+                        {"cost": [
+                          {"enemyDamage": [{
+                            "enemy": "Sidehopper",
+                            "type": "contact",
+                            "hits": 3
+                          }]},
+                          "Morph"
+                        ]}
+                      ]},
+                      {"and":[
+                        {"cost": [
+                          {"enemyDamage": [{
+                            "enemy": "Sidehopper",
+                            "type": "contact",
+                            "hits": 2
+                          }]},
+                          {"or":[
+                            "Spazer",
+                            "Charge",
+                            "Wave"
+                          ]}
+                        ]}
+                      ]},
+                      {"and":[
+                        {"cost": [
+                          {"enemyDamage": [{
+                            "enemy": "Sidehopper",
+                            "type": "contact",
+                            "hits": 1
+                          }]},
+                          {"or":[
+                            {"enemyKill":{
+                              "enemies": [
+                                ["Sidehopper"],
+                                ["Sidehopper"]
+                              ],
+                              "explicitWeapons": ["Missile", "PowerBomb"]
```

Comment:
Since the first Sidehopper can't reach you if you morph by the door after placing the PB some distance out the PowerBomb option should be 0 hits.

Reply:
I like it